client/my-sites/checkout/checkout-thank-you/redesign-v2/masterbar-styled/default-contact.tsx

```diff
@@ -28,6 +28,7 @@ const ContactContainer = styled.div`
 		text-decoration: underline;
 		line-height: 20px;
 		font-size: 14px;
+		padding: 0;
```

Comment:
| Before | After |
| - | - |
| ![header-before](https://github.com/Automattic/wp-calypso/assets/1101677/0c868951-040d-4a22-b95f-9f9f892c312a) | ![header-after](https://github.com/Automattic/wp-calypso/assets/1101677/363d0f24-a761-4f47-85b6-97fbae60ace3) |

There's a `.is-redesign-v2 .components-button` CSS rule that affects this button. Seeing as `Need extra help? Visit Help Center` reads as a single sentence, I didn't think this was intentional, which is why I reset this.

Reply:
#86271 will remove the `.is-redesign-v2 .components-button` rule, so I reverted this change.